content/en/Platform Deep Dive/Pentests/Findings/severity-levels.md

```diff
@@ -23,16 +24,14 @@ Risk = Likelihood * Impact
 
 In this case, the risk rating is based on the following factors:
 
-- Likelihood: Specifies the probability of exploiting the [finding](../getting-started/glossary/#finding).
+- **Likelihood**: Specifies the probability of exploiting the [finding](../getting-started/glossary/#finding).
 May include factors such as:
 
   - Skill required for an attacker to exploit a [vulnerability](../getting-started/glossary/#vulnerability)
   - Availability of documented exploits
   - Ease of exploiting the vulnerability
 
-<br>
-
-- Impact: Depends on the effect on technical and business operations. May include:
+- **Impact**: Depends on the effect on technical and business operations. May include:
```

Comment:
Given the label in the UI, should we change this to "Business Impact"?
<img width="239" alt="Screenshot 2022-12-12 at 6 38 48 AM" src="https://user-images.githubusercontent.com/84352037/207079577-8c6ffe04-caac-42af-a0c2-a818c24ed16a.png">

Reply:
I waffled on this one -- maybe the UI label is misleading?

Reply:
@mjang-cobalt Let's stick to the wording that OWASP is using: https://owasp.org/www-community/OWASP_Risk_Rating_Methodology

Risk = Likelihood * Impact